fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java

```diff
@@ -561,6 +561,11 @@ public TMasterOpResult proxyExecute(TMasterOpRequest request) {
 
         StmtExecutor executor = null;
         try {
+            // set session variables first
+            if (request.isSetModified_variables_sql()) {
+                LOG.info("Set session variables first: {}", request.modified_variables_sql);
```

Comment:
- you cannot correlate the original sql with this SetStatement.
- need info level log？

Reply:
1. Could explain why I cannot correlate the SQL with this SetStatement? I have already tested it, seems totally OK.
2. I think adding an info log here is more clear, some people may get confused about how those session variables are set. Besides, we only go to this branch if someone set a session variable, I suppose it won't happen very frequently?